pfhub/func.py

```diff
@@ -375,21 +414,25 @@ def compact(items):
     return filter(lambda x: x is not None, items)
 
 
-def read_vega_data(keys, data):
+def read_vega_data(keys, data, base_path):
     """Read vega data given keys to extract
 
     Read a vega data block given the keys (or columns) to extract
 
     Args:
       keys: columns of each data item
       data: the data block with the given columns
+      base_path: base path to use in case of relative links
 
     Returns:
       The data columns in a pandas DataFrame
 
     """
+    add_base_path = lambda x: x if isabs(x) else os.path.join(base_path, x)
```

Comment:
Consider using [pathlib](https://docs.python.org/3/library/pathlib.html)

Reply:
`isabs` checks if it's a URL as well.